docs/general/installation/truenas.md

```diff
@@ -0,0 +1,246 @@
+
+## First Steps
+
+The Jellyfin app in TrueNAS SCALE installs, completes the initial configuration, then starts the Jellyfin web portal.
+When updates become available, SCALE alerts and provides easy updates.
+
+You can configure environment variables at any time after deploying the application.
+
+SCALE does not need advance preparation.
+
+You can allow SCALE to create the datasets Jellyfin requires automatically during app installation.
+Or before beginning app installation, [create the datasets](https://www.truenas.com/docs/scale/scaletutorials/storage/datasets/datasetsscale/) to use in the **Storage Configuration** section during installation.
+Jellyfin requires two datasets: **config** and **cache**.
+You can organize these as one parent with two child datasets, for example mnt/tank/jellyfin/config, mnt/tank/jellyfin/cache, and so on.
+You can choose to create a static **transcodes** dataset or use temporary storage in the disk or memory for transcoding.
+
+If you want to run the application with a user or group other than the default apps (568) user and group, [create them](https://www.truenas.com/docs/scale/scaletutorials/credentials/managelocalusersscale/) now.
+
+## Installing the Jellyfin Application
+
+To install the **Jellyfin** application, go to **Apps**, click **Discover Apps**, either begin typing Jellyfin into the search field or scroll down to locate the **Jellyfin** application widget. You might need to click "Refresh Charts" if no Jellyfin app is showing in the results.
+
+![Jellyfin App Widget](/images/docs/install-truenas-1.png)
+
+Click on the widget to open the **Jellyfin** application details screen.
+
+![Jellyfin App Details Screen](/images/docs/install-truenas-2.png)
+
+Click **Install** to open the Jellyfin application configuration screen.
+
+Application configuration settings are presented in several sections, each explained below.
+To find specific fields click in the **Search Input Fields** search field, scroll down to a particular section or click on the section heading on the navigation area in the upper-right corner.
+
+![Install Jellyfin Screen](/images/docs/install-truenas-3.png)
+
+Accept the default values in **Application Name** and **Version**.
+
+Accept the defaults in **Jellyfin Configuration**, **User and Group Configuration**, and **Network Configuration** or change to suit your use case.
+You must select **Host Network** under **Network Configuration** if using [DLNA](https://jellyfin.org/docs/general/networking/dlna/).
+
+Jellyfin requires three app storage datasets.
+You can allow SCALE to create them for you, or use the dataset(s) created in [First Steps](#first-steps).
+Select the storage options you want to use for **Jellyfin Config Storage** and **Jellyfin Cache Storage**.
+Select **ixVolume (dataset created automatically by the system)** in **Type** to let SCALE create the dataset or select **Host Path** to use the existing datasets created on the system.
```

Comment:
![image](https://github.com/jellyfin/jellyfin.org/assets/25688628/5c612c61-cc9c-4b5f-aeda-eaa000d24ebd)
This section should be updated to match reality.

It would also be helpful to give hints on what types of storage this should be on.
For Cache and Config, these should preferably be on solid state storage, and shouldn't be on the same device as the media library if spinning storage is used.
For Transcodes, CMR HDDs can work.

Reply:
Will add in the storage type recommendations. 

I'm not sure I follow what about this you think does not match reality though?